doc/api/tls.markdown

```diff
@@ -542,9 +542,9 @@ Creates a new client connection to the given `port` and `host` (old API) or
 `options.port` and `options.host`. (If `host` is omitted, it defaults to
 `localhost`.) `options` should be an object which specifies:
 
-  - `host`: Host the client should connect to
```

Comment:
Let's keep this PR neat, please consider submitting this fixes in a separate pull request.